include/odp/api/spec/schedule_types.h

```diff
@@ -253,8 +255,8 @@ typedef struct odp_schedule_config_t {
 	 *  scheduling of the event and synchronization is maintained per flow
 	 *  within each queue.
 	 *
-	 *  Depeding on implementation, there may be much more flows supported
-	 *  than queues, as flows are lightweight entities.
+	 *  Depending on the implementation, there may be much more flows
+	 *  supported than queues, as flows are lightweight entities.
 	 *
```

Comment:
this typo correction could be moved to the group config patch.

Reply:
Fixed in V2.